src/nri/sampler.go

```diff
@@ -109,8 +110,8 @@ func (cs *ContainerSampler) SampleAll(ctx context.Context, i *integration.Integr
 		}
 
 		ms := entity.NewMetricSet(containerSampleName,
-			metric.Attr(attrContainerID, container.ID),
-			metric.Attr(attrShortContainerID, shortContainerID),
+			attribute.Attr(attrContainerID, container.ID),
```

Comment:
why this change is needed? due to the SDK upgrade or the cgroups?

Reply:
Yep, this was a change that happened in some minor of the sdk v3, `attribute` was moved to its own package.